gql/fragments/collections/AllFtvaFpb.gql

```diff
@@ -14,6 +14,7 @@ fragment AllFtvaFpb on ElementInterface {
     blocks: allFtvaFpb {
         id
         typeHandle
+        ftvaFlexibilePageBlockCardWithImageType
```

Comment:
Remove field `ftvaFlexibilePageBlockCardWithImageType` from this file and add it after this line https://github.com/UCLALibrary/ftva-website-nuxt/blob/main/gql/fragments/BlockCardWithImageFragment.gql#L5